src/main/java/com/legobmw99/allomancy/modules/powers/handlers/CommonEventHandler.java

```diff
@@ -274,13 +275,13 @@ public void onWorldTick(final TickEvent.WorldTickEvent event) {
                     /*********************************************
                      * BENDALLOY AND CADMIUM                     *
                      *********************************************/
-                    if (cap.isBurning(Metal.BENDALLOY) && !cap.isBurning(Metal.CADMIUM)) {
+                    if (!cap.isBurning(Metal.BENDALLOY) && cap.isBurning(Metal.CADMIUM)) {
```

Comment:
Making these effects not mutually exclusive is a poor design choice gameplay wise, and also just doesn't make sense lore wise

Reply:
Isn't Cadmium making everything else faster, and Bendalloy is making everything else slower?

Reply:
Yes, so using both at the same time doesn't really make sense in lore